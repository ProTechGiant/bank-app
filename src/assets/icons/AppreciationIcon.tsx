import Svg, { Path } from "react-native-svg";

import type { IconProps } from "@/assets/icons";

export function AppreciationIcon({ width = 20, height = 20, color = "#2E2E2E" }: IconProps) {
  return (
    <Svg width={width} height={height} viewBox="0 0 20 20" fill={color} xmlns="http://www.w3.org/2000/svg">
      <Path
        d="M1.25 3.75H12.9167L7.91667 11.25V14.5833H9.58333V16.25H4.58333V14.5833H6.25V11.25L1.25 3.75ZM8.83333 7.08333L10 5.41667H4.15833L5.325 7.08333H8.83333ZM14.5833 3.75H18.75V6.25H16.25V13.75C16.25 15.1333 15.1333 16.25 13.75 16.25C12.3667 16.25 11.25 15.1333 11.25 13.75C11.25 12.3667 12.3667 11.25 13.75 11.25C14.0417 11.25 14.325 11.3 14.5833 11.3917V3.75Z"
        fill={color}
      />
    </Svg>
  );
}
