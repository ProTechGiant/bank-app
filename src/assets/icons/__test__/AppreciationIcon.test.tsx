import { create } from "react-test-renderer";

import { AppreciationIcon } from "../AppreciationIcon";

describe("icon/AppreciationIcon", () => {
  it("renders a AppreciationIcon", () => {
    const result = create(<AppreciationIcon />);

    expect(result).toMatchSnapshot();
  });
});
