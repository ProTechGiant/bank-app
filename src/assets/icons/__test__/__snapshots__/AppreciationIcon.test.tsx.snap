// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Carousel renders a Carousel 1`] = `
<RNSVGSvgView
  align="xMidYMid"
  bbHeight="20"
  bbWidth="20"
  fill="#2E2E2E"
  focusable={false}
  height={20}
  meetOrSlice={0}
  minX={0}
  minY={0}
  style={
    [
      {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      {
        "flex": 0,
        "height": 20,
        "width": 20,
      },
    ]
  }
  vbHeight={20}
  vbWidth={20}
  width={20}
  xmlns="http://www.w3.org/2000/svg"
>
  <RNSVGGroup
    fill={
      {
        "payload": 4281216558,
        "type": 0,
      }
    }
    propList={
      [
        "fill",
      ]
    }
  >
    <RNSVGPath
      d="M1.25 3.75H12.9167L7.91667 11.25V14.5833H9.58333V16.25H4.58333V14.5833H6.25V11.25L1.25 3.75ZM8.83333 7.08333L10 5.41667H4.15833L5.325 7.08333H8.83333ZM14.5833 3.75H18.75V6.25H16.25V13.75C16.25 15.1333 15.1333 16.25 13.75 16.25C12.3667 16.25 11.25 15.1333 11.25 13.75C11.25 12.3667 12.3667 11.25 13.75 11.25C14.0417 11.25 14.325 11.3 14.5833 11.3917V3.75Z"
      fill={
        {
          "payload": 4281216558,
          "type": 0,
        }
      }
      propList={
        [
          "fill",
        ]
      }
    />
  </RNSVGGroup>
</RNSVGSvgView>
`;

exports[`icon/AppreciationIcon renders a AppreciationIcon 1`] = `
<RNSVGSvgView
  align="xMidYMid"
  bbHeight="20"
  bbWidth="20"
  fill="#2E2E2E"
  focusable={false}
  height={20}
  meetOrSlice={0}
  minX={0}
  minY={0}
  style={
    [
      {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      {
        "flex": 0,
        "height": 20,
        "width": 20,
      },
    ]
  }
  vbHeight={20}
  vbWidth={20}
  width={20}
  xmlns="http://www.w3.org/2000/svg"
>
  <RNSVGGroup
    fill={
      {
        "payload": 4281216558,
        "type": 0,
      }
    }
    propList={
      [
        "fill",
      ]
    }
  >
    <RNSVGPath
      d="M1.25 3.75H12.9167L7.91667 11.25V14.5833H9.58333V16.25H4.58333V14.5833H6.25V11.25L1.25 3.75ZM8.83333 7.08333L10 5.41667H4.15833L5.325 7.08333H8.83333ZM14.5833 3.75H18.75V6.25H16.25V13.75C16.25 15.1333 15.1333 16.25 13.75 16.25C12.3667 16.25 11.25 15.1333 11.25 13.75C11.25 12.3667 12.3667 11.25 13.75 11.25C14.0417 11.25 14.325 11.3 14.5833 11.3917V3.75Z"
      fill={
        {
          "payload": 4281216558,
          "type": 0,
        }
      }
      propList={
        [
          "fill",
        ]
      }
    />
  </RNSVGGroup>
</RNSVGSvgView>
`;
